src/main/java/seedu/address/logic/commands/AddPolicyCommand.java

```diff
@@ -67,7 +67,7 @@ public CommandResult execute(Model model) throws CommandException {
         model.setClient(clientToAddPolicy, addedPolicyClient);
         model.updateFilteredClientList(PREDICATE_SHOW_ALL_CLIENTS);
 
-        return new CommandResult(generateSuccessMessage());
+        return new CommandResult(generateSuccessMessage(), addedPolicyClient, true, false, false);
```

Comment:
I think we should standardize to select the client for all the commands, let's do this for v1.3b.